cypress/e2e/brewery/DatasetManager_Overview.cy.js

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import { Login, DatasetManager } from '../../commons/actions';
+import { TableParameters } from '../../commons/actions/generic';
```

Comment:
Could you please shorten the import to` import { TableParameters } from '../../commons/actions` :pray: :slightly_smiling_face: